src/web/src/app/space/list/component.ts

```diff
@@ -89,4 +93,13 @@ export class SpaceListComponent implements OnInit {
 		this.pageSize[id] = limit;
 		localStorage.setItem('spaceListPageSize', JSON.stringify(this.pageSize));
 	}
+
+	onRefresh() {
+		this.space.resetIsChecked();
+		this.space.Check().subscribe(
+			_ => of(true)
+		);
```

Comment:
А для чего такой трюк? Просто так не работает?
```suggestion
		this.space.Check().subscribe();
```